lib/src/models/song_recognition/index.dart

```diff
@@ -0,0 +1,2 @@
+export 'song_recognition.dart';
```

Comment:
Do we need this file? I think we could just export these individually in the `lib/src/models/models.dart` file.

Reply:
Won't it be too much of a cluster? We might have other models in the future, in this way we can separate by feature (through indexes)

Reply:
It's pretty much the same thing when you think about it. It's just removing the middleman.

Reply:
Yup but which one looks more ordered? or is more correct? or looks prettier? 😆  I can change it without a problem!